src/tests/test_apriori.cpp

```diff
@@ -318,4 +338,28 @@ TEST_F(ARAlgorithmTest, RepeatedExecutionConsistentResult) {
     }
 }
 
+TEST_F(ARAlgorithmTest, SupportAndConfidenceSingular) {
+    auto algorithm = CreateAlgorithmInstance(kRulesBook, 0.2, 0.5, 0, 1);
```

Comment:
I guess last parameter should be true

Reply:
No, it shouldn't. The last parameter is `itemColumnIndex`, which is `unsigned int`.